datacommons/statvar.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Google Inc.
```

Comment:
nit: ditto re: filename. maybe statvars or stat_vars or stats to mirror the other module names?

Reply:
Ok, I can make it plural